packages/frontend/src/components/DashboardTiles/TileForms/ChartUpdateModal.tsx

```diff
@@ -17,34 +18,32 @@ import { useChartSummaries } from '../../../hooks/useChartSummaries';
 import MantineIcon from '../../common/MantineIcon';
 
 interface ChartUpdateModalProps extends ModalProps {
-    chartTitle: string;
     onClose: () => void;
     onConfirm?: (newTitle: string, newChartUuid: string) => void;
+    tile: DashboardChartTile;
 }
 
 const ChartUpdateModal = ({
-    chartTitle,
     onClose,
     onConfirm,
+    tile,
     ...modalProps
 }: ChartUpdateModalProps) => {
-    const form = useForm({
-        initialValues: {
-            uuid: '',
-            title: '',
-        },
-    });
+    const form = useForm({});
     const { projectUuid } = useParams<{ projectUuid: string }>();
     const { data: savedCharts, isLoading } = useChartSummaries(projectUuid);
 
-    const handleConfirm = form.onSubmit(
-        ({ title: newTitle, uuid: newChartUuid }) => {
-            onConfirm?.(newTitle, newChartUuid);
-        },
+    const [chartUuid, setChartUuid] = React.useState<string>(
+        tile.properties.savedChartUuid ?? '',
+    );
+    const [chartTitle, setChartTitle] = React.useState<string>(
```

Comment:
Why can't we track these input values in the useForm?

Reply:
we need the input fields to be controlled so we can populate them